pandas/_libs/parsers.pyx

```diff
@@ -255,7 +255,7 @@ cdef extern from "parser/tokenizer.h":
 
 #    inline int to_complex(char *item, double *p_real,
 #                          double *p_imag, char sci, char decimal)
-    inline int to_longlong(char *item, long long *p_value) nogil
```

Comment:
actually this line doesn't appear to be used anymore, so I think can comment out

Reply:
OK.  I've got an upcoming PR that this will be appropriate for.  Is there a pattern for when to comment out vs delete?

Reply:
these should all be deleted